test/connectors/BusinessCustomerConnectorSpec.scala

```diff
@@ -20,43 +20,42 @@ import audit.Auditable
 import builders.AuthBuilder
 import config.ApplicationConfig
 import models._
-import org.mockito.{ArgumentMatchers, MockitoSugar}
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito._
 import org.scalatest.BeforeAndAfterEach
 import org.scalatestplus.play.PlaySpec
 import org.scalatestplus.play.guice.GuiceOneServerPerSuite
 import play.api.libs.json.{JsValue, Json}
 import play.api.test.Helpers._
+import uk.gov.hmrc.connectors.ConnectorTest
 import uk.gov.hmrc.http._
 import uk.gov.hmrc.play.audit.DefaultAuditConnector
-import uk.gov.hmrc.play.bootstrap.http.DefaultHttpClient
 
 import java.util.UUID
 import scala.concurrent.{ExecutionContext, Future}
 
-class BusinessCustomerConnectorSpec extends PlaySpec with GuiceOneServerPerSuite with MockitoSugar with BeforeAndAfterEach {
+class BusinessCustomerConnectorSpec extends PlaySpec with GuiceOneServerPerSuite with ConnectorTest with BeforeAndAfterEach {
```

Comment:
ConnectorTest should be used on each test rather than the Spec class to ensure no interaction between tests

Reply:
Resolved.